docs/configuration.md

```diff
@@ -663,24 +663,27 @@ Filters low-value coins which would not allow setting stoplosses.
 #### PriceFilter
 
 The `PriceFilter` allows filtering of pairs by price. Currently the following price filters are supported:
+
 * `min_price`
 * `max_price`
 * `low_price_ratio`
 
 The `min_price` setting removes pairs where the price is below the specified price. This is useful if you wish to avoid trading very low-priced pairs.
-This option is disabled by default, and will only apply if set to <> 0.
+This option is disabled by default (or set to 0), and will only apply if set to > 0.
```

Comment:
I'm not sure this "(or set to 0)" makes much sense ... 

either it's disabled - or set to 0 by default - but the documentation should know (this sounds like a question to me ...

Reply:
I agree with xmatthias, reads better without the "(or set to 0)".